src/mame/barcrest/mpu3.cpp

```diff
@@ -747,14 +746,14 @@ void mpu3_state::machine_start()
 }
 
 /* generate a 50 Hz signal (some components rely on this for external sync) */
-TIMER_DEVICE_CALLBACK_MEMBER(mpu3_state::gen_50hz)
+TIMER_DEVICE_CALLBACK_MEMBER(mpu3_state::gen_100hz)
 {
 	/* Although reported as a '50Hz' signal, the fact that both rising and
 	falling edges of the pulse are used means the timer actually gives a 100Hz
 	oscillating signal.*/
-	m_signal_50hz = m_signal_50hz?0:1;
-	m_ptm2->set_c1(m_signal_50hz);
-	m_pia3->cb1_w(~m_signal_50hz);
+	m_signal_100hz = m_signal_100hz?0:1;
```

Comment:
While you're here, may as well clean it up a little: "m_signal_100hz ^= 1" would work to toggle it without needing the compare.